blog/2020-10-23-supervisor-update.md

```diff
@@ -0,0 +1,90 @@
+- Using `X-Hassio-Key` header as authentication method is deprecated, and will be removed later this year. You need use an authorization header with a `Bearer` token instead.
+
+The [API documentation][api_docs] have been updated to reflect these changes.
+
+## Add-on options
+
+The permissions of the `/data/options.json` file, was change from `644` to `600`. If your add-on is running as non-root and you are reading this file, it will give now you permission issues.
+
+There are several steps you can do in your add-on to continue to use this information:
+
+- If you are using [S6-overlay] in your add-on, you can use [`/etc/fix-attrs.d`][S6-overlay-permissions] to ensure that the user you are running the add-on as, have access to the file.
+- You can change your add-on to run as `root` (default)
+
+## Releases
+
+Until now, the Supervisor, our plugins and add-ons have been using a mix of the build number and [Semantic Versioning (SemVer)][semver] as the versioning system, we have decided to replace that for these repositories and to adopt [Calendar Versioning (CalVer)][calver] as our versioning system instead.
+
+We are migrating the Supervisor from release based development to continuous development. This fits perfectly with our exists channel based update strategy (stable, beta and dev). We are now leveraging automated pipelines, to build and push out new Supervisor versions to the correct channels. By moving to this structure there was no more need to a dual branch setup, so both our `dev` and `master` branches have now been replaced with a new `main` branch. Our plugins (DNS, Multicast, Observer, CLI) for the Supervisor will also follow this continuous development principal.
+
+We did this move to provide higher software quality with automatic test system. Every commit now trigger a new dev release which get tested by our test instances and issues are imminently reported to sentry, this gives us the opportunity to test all changes before we create a releases, when a release is created the the changes are available to the beta channel. Based on the feedback and reports to Sentry we can decide to push this into stable channel.
+
+We are using [Azure pipelines][azure_pipelines] to build and publish the Supervisor. If you are interested on how we are doing this, you can look at the [release pipeline here][release_pipeline], and the [template repository here][azure_templates].
+
+## GitHub Action
+
+You can now use our [builder][builder] as a [GitHub action][github_action]! :tada:
+
+This is already in use in our [hassio-addons repository][addons], you can see an example on how we implemented it [here][builder-action].
+
+It can be used both to ensuring that the add-on will still build with changes made to your repository, and to publish the images as a part of a release workflow. How to use the action is documented in the [builder repository][builder-action].
```

Comment:
```suggestion
It can be used both to ensure that the add-on will still build with changes made to your repository, and to publish the images as part of a release workflow. How to use the action is documented in the [builder repository][builder-action].
```